### ISSUE TITLE
Support for `production` & `profiling` React builds

### ISSUE THREAD

**Issue by dinkinflickaa** [lahmatiy/react-render-tracker#25]
I am trying to run [react-render-tracker](https://github.com/lahmatiy/react-render-tracker) against a react profiling build but seeing this error in the console

```
dispatcher-trap.ts:292 
       Uncaught (in promise) TypeError: renderer.getCurrentFiber is not a function
    at Object.set [as current] (dispatcher-trap.ts:292:41)
    at li (react-dom.profiling.min.js:169:287)
    at xc (react-dom.profiling.min.js:169:424)
    at sf (react-dom.profiling.min.js:163:248)
    at Qa (react-dom.profiling.min.js:158:295)
    at Hd (react-dom.profiling.min.js:195:178)
    at react-dom.profiling.min.js:198:188
    at qi (react-dom.profiling.min.js:164:220)
    at Id (react-dom.profiling.min.js:198:174)
    at Object.N.render (react-dom.profiling.min.js:250:154)
```

Are there any plans to make the tracker compatible with the profiling build?

**Comment by lahmatiy** [comment]
@dinkinflickaa Production and profiling builds provide limited API to React's internals. So, some data can't be captured for such build. I have plans to support such builds with limited functionality. However, there is no ETA at the moment, since I focused on other features for now.

It also surprises me that you are getting an exception instead of a warning: "[react-render-tracker] Unsupported React renderer, only bundle type 1 (development) is supported but 0 (production) is found". What versions of React Render Tracker and React are you using?

**Comment by dinkinflickaa** [comment]
Thanks for your response @lahmatiy. I made some changes locally to run the tracker against a profiling build, The stacktrace is probably the side effect of that.

**Comment by lahmatiy** [comment]
I found that the new check and warning was not published at the moment of writing my comment. So now you should see a warning in RRT about an unsupported build.

**Comment by lahmatiy** [comment]
Time to time, I get requests to support React build type other than `development` (i.e. `production` & `profiling`). Technically, it's possible but with a drawback that RRT will capture less data in this case, e.g. no owner-based hierarchy of components and less precision for a render/update cause detection. However, in some cases it might be not important at all. So I going to add such a support, with a reduction in data details due to a build type restrictions.

**Comment by lahmatiy** [comment]
Just released [0.7.1](https://github.com/lahmatiy/react-render-tracker/releases/tag/v0.7.1) with basic support for build & profile React builds.
Due to lack of API provided by build & profile builds at the moment, some data is not capturing for these builds:
- No owner for components, so only parent-child hierarchy in the components tree is supported. It also prevents detection of triggers for an update in most cases
- No info about used hooks, including used contexts on functional components, memo hooks recomputation etc.
- Some updates might be missed in the event log
